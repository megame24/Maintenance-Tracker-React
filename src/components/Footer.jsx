import React from 'react';
import { Link } from 'react-router-dom';

const Footer = () => (
  <footer>
    <p>
      &copy; Maintenance Tracker
      &nbsp; &nbsp;
      <Link to="/about">About Us</Link>
    </p>
  </footer>
);


export default Footer;
