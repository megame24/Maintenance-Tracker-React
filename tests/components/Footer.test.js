import React from 'react';
import { shallow } from "enzyme";
import Footer from '../../src/components/Footer';

describe('Testing Footer component', () => {
  it('should render as expected', () => {
    const tree = shallow(<Footer />);
    expect(tree).toMatchSnapshot();
  });
});
