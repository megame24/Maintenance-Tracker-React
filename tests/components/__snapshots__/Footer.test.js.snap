// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing Footer component should render as expected 1`] = `
<footer>
  <p>
    © Maintenance Tracker    
    <Link
      replace={false}
      to="/about"
    >
      About Us
    </Link>
  </p>
</footer>
`;
